src/SDKs/HDInsight/Management/Management.HDInsight/Management.HDInsight.csproj

```diff
@@ -7,11 +7,11 @@
 		<PackageId>Microsoft.Azure.Management.HDInsight</PackageId>
 		<Description>Azure HDInsight Management SDK Library</Description>
 		<AssemblyName>Microsoft.Azure.Management.HDInsight</AssemblyName>
-		<Version>4.2.0-preview</Version>
+		<Version>4.2.1-preview</Version>
```

Comment:
Since this is a breaking change, please revert this to `4.3.0-preview`. We need to revisit the versioning when you wish to move to the next API version or when you release the next stable version